### ISSUE TITLE
Server doesn't work

### ISSUE THREAD

**Issue by romainl** [vim-advent-calendar/vimways.org#71]
I was unable to get the server commands to work

    $ hugo server -DF

results in:

    ERROR 2019/08/28 11:25:51 Failed to add template
    "_default/list.html" in path "_default/list.html": template: _default/list.html:12: unexpected "=" in operand
    Total in 224 ms
    Error: Error building site: logged 1 error(s)

**Comment by alerque** [comment]
Works for me. I just installed Hugo (0.57.2 extended), cloned this repository, and ran the command. Are you sure you have Hugo built with the `extended` option (or installed from a package manager that enabled them)?

**Comment by shlomif** [comment]
I still think we should have CI to avoid the "works for me" syndrome. See: https://github.com/shlomif/Freenode-programming-channel-FAQ/blob/master/FAQ_with_ToC__generated.md#what-do-continuous-integration-ci-services-such-as-travis-ci-jenkins-or-appveyor-provide and https://github.com/shlomif/testing-static-sites . I can try working on it.

**Comment by romainl** [comment]
The template makes use of the new-ish `=` operator

https://gohugo.io/templates/introduction/#variables

Looks like you are using an older Hugo. [As stated in the `README`](https://github.com/vim-advent-calendar/vimways.org#requirements) you should use "0.55 extended" or newer.

**Comment by romainl** [comment]
@shlomif I tend to agree on the need for CI but the issue at hand is on a user's machine where no amount of CI could be of help.
